GitCommands/Settings/BuildServerSettings.cs

```diff
@@ -1,40 +1,46 @@
 ﻿using GitUIPluginInterfaces;
-using GitUIPluginInterfaces.Settings;
+using GitUIPluginInterfaces.BuildServerIntegration;
+using Microsoft;
 
 namespace GitCommands.Settings
 {
     internal sealed class BuildServerSettings : IBuildServerSettings
     {
         private const string BuildServerGroupName = "BuildServer";
 
-        private const string? TypeDefault = null;
-        private const bool EnableIntegrationDefault = false;
-        private const bool ShowBuildResultPageDefault = true;
-
         private readonly ISettingsSource _settingsSource;
 
         public BuildServerSettings(ISettingsSource settingsSource)
         {
             _settingsSource = settingsSource;
         }
 
-        public string? Type
+        /// <inheritdoc />
+        public ISettingsSource GetSettings()
+        {
+            string? type = BuildServerType;
+            Validates.NotNull(type);
```

Comment:
This is actually incorrect. `type` can be null, e.g. when no prior build server configuration is present. To repro remove/rename all GitExtensions.settings files at all levels (global, team, local).